test/e2e/tests/ppom/ppom-blockaid-alert-simple-send.spec.js

```diff
@@ -127,7 +127,10 @@ async function mockInfuraWithFailedResponses(mockServer) {
  */
 describe('Simple Send Security Alert - Blockaid @no-mmi', function () {
   // eslint-disable-next-line mocha/no-skipped-tests
-  it.skip('should not show security alerts for benign requests', async function () {
+  it('should not show security alerts for benign requests', async function () {
+    if (process.env.MULTICHAIN) {
+      return;
+    }
```

Comment:
the flow for the multichain build needs to be updated altogether. It's left out of scope of this PR, since this purely focuses on enabling mocks. The update multichain flow should be tackled in a subsequent PR (see description)